website/astro.config.ts

```diff
@@ -1,40 +1,9 @@
-import { netlifyStatic } from "@astrojs/netlify";
```

Comment:
The static Netlify adapter will be deprecated, so I've removed it. Note that we technically don't even need this one, so we should feel safe to remove it as well.

Reply:
We don't??? 😯🤯

Reply:
Nope, both Vercel and Netlify let you deploy just fine without an adapter, we do mention it in the docs: https://docs.astro.build/en/guides/integrations-guide/netlify/

It might enable some host-specific features but for the most part, the adapters aren't focused on the static use case.